### PR TITLE
Adding data

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
guys dont worry about number of files, its just image files 
image

### CODE REVIEW

**Review by muskaanshah — APPROVED** [review]
Looks good, I like your folder structure - very organised

File: my-app/src/context/data/DataContext.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            try {
+                const {data}=await axios.get("/api/products")
+
+                setProducts(prev=>[...prev,...data.products])
+            } catch (error) {
+                console.log(error)
+            }
+        })();
+    }, [])
+    // console.log(products)
```
Comment (manismk) [inline my-app/src/context/data/DataContext.jsx:23]: You can remove the commented code

**Review by manismk — APPROVED** [review]
Looks good. Good folder structure.

File: my-app/src/context/data/DataContext.jsx (hunk 1 above)
Comment (mohammadkhizerkhan) [reply my-app/src/context/data/DataContext.jsx:23]: sure

**Pull request merged**